notification-service/src/main/resources/application.yml

```diff
@@ -25,19 +18,20 @@ spring:
             enable: true
             required: true
     verify:
-      host: ${VERIFY_EMAIL_HOST}
+      host: http://localhost:587
 
 kafka:
+  broker:
+    host: localhost:9092
   topics:
-    notification-topic: notification-topic
-    account-topic: account-topic
+    notification-topic:
+      name: notification-topic
+      group-id: group-id
+    account-topic:
+      name: account-topic
+      group-id: group-id-2
 
-server:
-  port: 8083
+account-confirmation-url: "http://localhost:8080/auth/confirm?token="
```

Comment:
Values that change depending on the environment (in this case: host, port) should be moved to be in the .env file.

Reply:
Done.